homeassistant/components/opower/config_flow.py

```diff
@@ -23,6 +23,7 @@
         vol.Required(CONF_UTILITY): vol.In(get_supported_utility_names()),
```

Comment:
See comment in https://github.com/tronikos/opower/pull/13 for adding a parameter here to be backwards compatible.